app/code/Magento/Search/Test/Mftf/Test/AdminNewSearchSynonymsFormResetTest.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    /**
+    * Copyright © Magento, Inc. All rights reserved.
+    * See COPYING.txt for license details.
+    */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminNewSearchSynonymsFormResetTest">
+        <annotations>
+            <features value="Search"/>
+            <stories value="Reset new search synonyms group form"/>
+            <title value="Admin reset new search synonyms group form"/>
+            <description value="When admin users reset button on new search synonyms form all fields should be set to default values"/>
+            <severity value="AVERAGE"/>
+            <group value="Search"/>
+        </annotations>
+        <before>
+            <actionGroup ref="AdminLoginActionGroup" stepKey="LoginAsAdmin"/>
+        </before>
+        <after>
+            <actionGroup ref="AdminLogoutActionGroup" stepKey="logout"/>
+        </after>
+
+        <!--TEST BODY -->
```

Comment:
Please, remove comments from the test body. In this particular case, the comments don't provide important information but will be picked up by a test runner as separate test steps.

Reply:
@rogyar I have removed comments. can you please review now?